src/commcare_cloud/commands/ansible/ansible_playbook.py

```diff
@@ -337,6 +337,27 @@ def run(self, args, unknown_args):
         return AnsiblePlaybook(self.parser).run(args, unknown_args)
 
 
+class AddNewUser(_AnsiblePlaybookAlias):
+    command = 'add-new-user'
```

Comment:
nit, but can this be `update-user` or `add-user` to denote that it's running `update-users` on a select group. Semantically, `update-users` can also add new users, so removing `new` from it makes it simpler.